Makefile

```diff
@@ -59,15 +60,33 @@ release-install: release-container
 
 verify-shell: docker-running
 	docker run -ti --rm \
-		--privileged \
 		--volume /home/cockpit:/home/user \
 		--volume $(CURDIR)/verify:/usr/local/bin \
 		--volume=/opt/verify:/build:rw \
-		--net=host --pid=host --privileged --entrypoint=/bin/bash \
+		--net=host --pid=host --entrypoint=/bin/bash \
```

Comment:
Was the intent really to drop *both* `--privileged` options, or just drop the redundant one? When dropping both, I think that at least warrants a detailled commit message or a separate commit. Fine for me to do an en passant cleanup of just one of them.